README.md

```diff
@@ -78,6 +88,17 @@ The Simplify jar will be in `simplify/build/libs/simplify.jar`. You can test it'
 java -jar simplify/build/libs/simplify.jar -it 'org/cf' simplify/obfuscated-example
 ```
 
+**Windows users**
+
+Using CMD:
+
+```
+cd [location of your simplify folder]
+gradlew fatjar
+```
+
+If build fails because gradlew is looking for JDK tools in your JRE, close the CMD, add in system environnement variables "JAVA_HOME" as new variable name and your JDK path as value, then reproceed.
```

Comment:
This should be moved to the troubleshooting section. I think it'd be good to contain the exact error because a user might not know that know that "missing tools.jar" means gradle is looking in the wrong Java path, so they wouldn't know this advice applies to them.

Reply:
I thought about that also, but the troubleshoot section doesn't treat of building error. But well, will change it.

Reply:
I'm OK relegating this to it's own section because the readme should be really crisp and concise and I don't think this is a common error. I think it's pretty basic Java dev stuff.

Reply:
Alright, so I will move it to the troubleshooting session.